app/src/main/res/layout/fragment_user_details.xml

```diff
@@ -30,7 +30,7 @@
             app:layout_constraintTop_toTopOf="@+id/image_blur" />
 
         <ImageView
-            android:id="@+id/image_avatar"
+            android:id="@+id/user_image_avatar"
```

Comment:
Please make sure to follow our coding style before changing the things! Wha's the reason to change the id of the imageView here?